src/game/AI/ScriptDevAI/scripts/kalimdor/dustwallow_marsh.cpp

```diff
@@ -32,11 +32,24 @@ at_nats_landing
 boss_tethyr
 EndContentData */
 
+#include "AI/BaseAI/UnitAI.h"
```

Comment:
dont add includes before sc_common.h, thats the PCH one and it can mess with order. Also, if you include CombatAI, UnitAI is included there.

Reply:
The includes are as always automatically added by my editor and will be removed with my final commit